pkg/fribidi

```diff
@@ -1,13 +1,17 @@
 [mirrors]
-http://fribidi.org/download/fribidi-0.19.7.tar.bz2
+https://github.com/fribidi/fribidi/archive/v1.0.5.tar.gz
 
 [vars]
-filesize=648299
-sha512=4bfdda43d961030a8a2ae2c98d2ca5d43c1b59633fa0987920c5537ec69f3f624204ceacc4c474d878cbb417ee2d9e708dfe169cfce6ad584d2f2d5122678b63
+filesize=2086906
+sha512=c51b67cc3e7610bef9a66f2456f7602fe010164c2c01e7d91026cefa4a08fdce5165b6eb3814e76cd89e766356fb71adc08bf75d9b2f5802f71c18b5d0476887
+tarball=fribidi-1.0.5.tar.gz
+pkgver=2
 
 [deps]
 
 [build]
+autoreconf --force --install --verbose
```

Comment:
sh*t... don't they have proper release tarballs shipping pre-built configure script ?
we pretty much dislike pulling in [deps.host] for autoconf and automake :/
i think we do it only in very few packages

Reply:
maybe an older version has them preshipped - I'll check.